### ISSUE TITLE
Directives and pipes are not recognized from a library

### ISSUE THREAD

**Issue by arobinson** [angular/vscode-ng-language-service#1083]
**Describe the bug**
Using a local Ivy library in a mono-repo style project setup, the Angular Language Service does not recognize the pipe nor the directive.

**To Reproduce**
1. Clone https://github.com/arobinson/ALSDirectivePipeIssue
2. `cd ALSDirectivePipeIssue/ExampleApp`
3. `npm i`
4. `ng b`
5. `code ..` (open VS code to the root folder of the workspace)
6. Open `app.component.html` in VS code
7. Command+hover over the `libExampleDirective` and the `examplePipe`
8. Notice that there is no support for them

**Expected behavior**

Should provide pipe and directive help and go to source help of an Ivy library

**Logs**

Please attach two logs:

1. Console output (Go to toolbar --> View --> Output, and copy the contents) → None
2. Log file (Path is printed to the console output at startup):
```
[Info  - 2:49:37 PM] Angular language server process ID: 53643
[Info  - 2:49:37 PM] Using typescript/lib/tsserverlibrary v4.1.3 from /Users/andrew/.vscode/extensions/angular.ng-template-11.1.0/node_modules/typescript/lib/tsserverlibrary.js
[Info  - 2:49:37 PM] Using @angular/language-service/bundles/ivy v11.1.0 from /Users/andrew/.vscode/extensions/angular.ng-template-11.1.0/node_modules/@angular/language-service/bundles/ivy.js
[Info  - 2:49:37 PM] Log file: /Users/andrew/Library/Application Support/Code/logs/20210125T144210/exthost1/Angular.ng-template/nglangsvc.log
[Info  - 2:49:40 PM] Enabling Ivy language service for /Users/andrew/development/work/Angular/DirectiveIssue/ExampleApp/tsconfig.json.
[Error - 2:49:40 PM] Error: Symbol ExampleLibraryModule declared in /Users/andrew/development/work/Angular/DirectiveIssue/ExampleLibraries/dist/example-library/lib/example-library.module.d.ts is not exported from example-library (import into /Users/andrew/development/work/Angular/DirectiveIssue/ExampleApp/src/app/app.module.ts)
[Info  - 2:49:40 PM] Enabling Ivy language service for /Users/andrew/development/work/Angular/DirectiveIssue/ExampleLibraries/tsconfig.json.
```

**Screenshots**

![image](https://user-images.githubusercontent.com/93213/105770639-b8dac480-5f1c-11eb-9d43-1f0660f95b16.png)

**Additional context**
- VS code Version: 1.52.1
- Angular 11.0.5
- Angular Language Service 11.1.0
- Mac OSX 10.15.7

**Comment by kyliau** [comment]
Stack trace:
```
Error: Symbol ExampleLibraryModule declared in ALSDirectivePipeIssue/ExampleLibraries/dist/example-library/lib/example-library.module.d.ts is not exported from example-library (import into ALSDirectivePipeIssue/ExampleApp/src/app/app.module.ts)
    at AbsoluteModuleStrategy.emit (node_modules/@angular/language-service/bundles/ivy.js:21607:23)
    at ReferenceEmitter.emit (node_modules/@angular/language-service/bundles/ivy.js:21528:42)
    at toR3Reference (node_modules/@angular/language-service/bundles/ivy.js:27289:33)
    at NgModuleDecoratorHandler._toR3Reference (node_modules/@angular/language-service/bundles/ivy.js:29713:24)
    at node_modules/@angular/language-service/bundles/ivy.js:29535:56
    at Array.map (<anonymous>)
    at NgModuleDecoratorHandler.analyze (node_modules/@angular/language-service/bundles/ivy.js:29535:40)
    at TraitCompiler.analyzeTrait (node_modules/@angular/language-service/bundles/ivy.js:25340:40)
    at analyze (node_modules/@angular/language-service/bundles/ivy.js:25306:44)
    at TraitCompiler.analyzeClass (node_modules/@angular/language-service/bundles/ivy.js:25328:21)"
```

**Comment by kyliau** [comment]
I forgot to implement `realpath()` in the language service adapter, and as a result symlinks cannot be resolved properly. Hoping to get a fix merged and released today! Thanks a lot for the repro, appreciate it.

**Comment by kyliau** [comment]
Fix released in [v11.1.1](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.1).

**Comment by arobinson** [comment]
Thank you!
